distribution/openhabhome/configurations/transform/miosShutterCommand.map

```diff
@@ -0,0 +1,4 @@
+DOWN=urn:upnp-org:serviceId:WindowCovering1/Down()
```

Comment:
The content looks good, but since they're MiOS specific I've put these into the directory:
```
bundles/binding/org.openhab.binding.mios/examples/transform/
```

and let people copy them over (manually) as needed.